server/models/Request.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
+  {
+    user_id: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    sitter_id: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    start_date: {
+      type: Date,
+      required: true,
+    },
+    end_date: {
+      type: Date,
+      required: true,
+    },
+    accepted: {
+      type: Boolean,
+      default: false,
+    },
+    declined: {
+      type: Boolean,
+      default: false,
+    },
+    paid: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  { timestamps: true }
+);
+
+module.exports = Request = mongoose.model("request", requestSchema);
```

Comment:
Mongoose models are named in `PascalCase` by convention so let's adjust this accordingly.

Reply:
Updated